config/hubs/carbonplan-azure.cluster.yaml

```diff
@@ -0,0 +1,190 @@
+name: carbonplan-azure
+provider: kubeconfig
+kubeconfig:
+  file: secrets/carbonplan-azure.yaml
+support:
+  config:
+    prometheus:
+      server:
+        resources:
+          requests:
+            cpu: 1
+            memory: 4Gi
+          limits:
+            cpu: 4
+            memory: 8Gi
+    grafana:
+      ingress:
+        hosts:
+          - grafana.carbonplan-azure.2i2c.cloud
```

Comment:
Can we instead use grafana.azure.carbonplan.2i2c.cloud instead? And the hub can be at azure.carbonplan.2i2c.cloud? That way, if we do GCP, we can do gcp.carbonplan.2i2c.cloud, and ideally I guess the AWS one could've been at aws.carbonplan.2i2c.cloud

Reply:
Done in [`8d33c51`](https://github.com/2i2c-org/infrastructure/pull/838/commits/8d33c519ce8b540e7f0600247485cefdbdf6ddc0)